jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/junit4/EtcdClusterResource.java

```diff
@@ -14,15 +14,23 @@
  * limitations under the License.
  */
 
-package io.etcd.jetcd.launcher.junit;
+package io.etcd.jetcd.launcher.junit4;
```

Comment:
are you plan to remove this file? since we are probably not going to use the junit4 version.

Reply:
I think it makes sense to leave it for https://github.com/etcd-io/jetcd/issues/546. So people who are using JUnit4 could still use this for testing.

Reply:
fair enough

Reply:
Also it is used in `PaxExamWrapperTest` in `jetcd-osgi`